### ISSUE TITLE
ImportError: No module named 'simdata'

### ISSUE THREAD

**Issue by sksq96** [jbrambleDC/simulacram#7]
The following module is not installed along with `simulacrum`, neither is it available at pip.

```
>>> import simulacrum as sm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/shubham/.pyenv/versions/anaconda3-4.1.1/lib/python3.5/site-packages/simulacrum/__init__.py", line 8, in <module>
    from simdata import *
ImportError: No module named 'simdata'
```

**Comment by jbrambleDC** [comment]
Hi I have addressed this in #8 try to install as dev, and then let me know if that fixes your issue on python 3.5? @sksq

**Comment by jbrambleDC** [comment]
closing, reopen if whats currently on master doesn't solve your issues!
